src/components/delegates/delegates.js

```diff
@@ -39,6 +39,7 @@ app.component('delegates', {
       this.votedList = [];
       this.unvoteList = [];
       this.loading = true;
+      this.$scope.$emit('showLoadingBar');
```

Comment:
repeating the same event emit and the manipulating a flag to manage the behaviour is not a good approach. but since it's temporary I'll approve it.